### PR TITLE
Tidy testing framework code

### PR DESCRIPTION
**Pull request by GRyall**
Generally tidy up the test suite a bit:
* Removes a quantity of commented out redundant code from across the test
suite
* Ensure uniform indenting across code within the test suite
* Tidy up some comments
* Change the way some functions are spread across lines
* Remove some empty lines
* Add some curly braces to some if statements (the code works without them, but this improves readability)
* Fix a typo in a comment in one of the unit tests

### CODE REVIEW

File: tests/doctrine/DoctrineCleanInsert1Test.php [hunk 1]
```diff
@@ -1,12 +1,7 @@
 <?php
-
-//require_once 'PHPUnit/Extensions/Database/TestCase.php';
-//require_once 'PHPUnit/Extensions/Database/DataSet/DefaultDataSet.php';
 require_once dirname(__FILE__) . '/TestUtil.php';
-
-use Doctrine\ORM\EntityManager;
 require_once dirname(__FILE__) . "/bootstrap.php";
-
+use Doctrine\ORM\EntityManager;
```
Comment (jrha) [inline tests/doctrine/DoctrineCleanInsert1Test.php]: So this line moved?

Comment (GRyall) [reply tests/doctrine/DoctrineCleanInsert1Test.php]: For neatness I moved it after the requires, partly to check my understanding that the order wouldn't matter. That said, I didn't do it on other files, so am happy to undo this if needed

Comment (jrha) [reply tests/doctrine/DoctrineCleanInsert1Test.php]: Just wanted to make sure it was intentional!

File: tests/unit/lib/Gocdb_Services/RoleActionAuthorisationServiceTest.php [hunk 2]
```diff
@@ -16,957 +16,911 @@
...
+    $this->assertTrue(in_array($ngiManagerRole, $enablingRoles));
+    $this->assertTrue(in_array($rodUserRole, $enablingRoles));
+
+    // Delete the user's Project COD role
+    $this->em->remove($codRole);
+    $this->em->flush();
+    /*
+    *                               p1
+    *                                |
+      *                             ngi      <- ngiManRT, rodRT,
```
Comment (gregcorbett) [inline tests/unit/lib/Gocdb_Services/RoleActionAuthorisationServiceTest.php]: extra indentation here?

**Comment by gregcorbett** [comment]
Other than my comment about one line of indentation, LGTM :)

File: tests/unit/lib/Gocdb_Services/RoleActionAuthorisationServiceTest.php (hunk 2 above)
Comment (GRyall) [reply tests/unit/lib/Gocdb_Services/RoleActionAuthorisationServiceTest.php]: Fixed in additional commit

**Pull request merged**